pkg/schemadsl/parser/tests/complexcaveat.zed

```diff
@@ -0,0 +1,14 @@
+definition another {}
+
+caveat somecaveat(somecondition uint, somebool bool) {
```

Comment:
may be worth using our own custom types as well

Reply:
From a lexer and parser perspective, there isn't a difference, so I don't think we need to model that here